### PR TITLE
Remove logs, metrics, and uptime content

### PR DESCRIPTION
**Pull request by EamonnTP**
This PR removes the `logs`, `metrics`, and `uptime` directories and its content from the `master`, `7.x`, and `7.10` branches of this repo. 

The content for the Logs, Metrics, and Uptime apps has now moved into the `docs/en/observability` directory.

### Related PR

https://github.com/elastic/docs/pull/1985

### Related issue

Closes https://github.com/elastic/observability-docs/issues/189

### CODE REVIEW

**Comment by EamonnTP** [comment]
@bmorelli25 Is the doc build failure expected until I update the `conf.yaml` file in the `docs` repo?

**Comment by bmorelli25** [comment]
Correct. This is blocked until elastic/docs#1985 is merged

**Comment by EamonnTP** [comment]
@elasticmachine, run elasticsearch-ci/docs

**Review by bmorelli25 — APPROVED** [review]

**Pull request merged**
